spring.go

```diff
@@ -0,0 +1,405 @@
+package beego
+
+import (
+	"errors"
+	"github.com/astaxie/beego/context"
+	"reflect"
+	"regexp"
+	"strings"
+)
+
+const (
+	TagName = "autowired"
```

Comment:
warning: exported const TagName should have comment (or a comment on this block) or be unexported (golint)